CI: v1.12: Conformance AKS fails during provisioning: UnsupportedAliasMinorVersion

## CI failure

```
Run # Create group
{
  "id": "/subscriptions/***/resourceGroups/cilium-cilium-5825160549-2-1",
  "location": "westeurope",
  "managedBy": null,
  "name": "cilium-cilium-5825160549-2-1",
  "properties": {
    "provisioningState": "Succeeded"
  },
  "tags": {
    "owner": "27[40](https://github.com/cilium/cilium/actions/runs/5825160549/job/15798796710#step:10:41)6",
    "usage": "cilium-cilium"
  },
  "type": "Microsoft.Resources/resourceGroups"
}
WARNING: The behavior of this command has been altered by the following extension: aks-preview
WARNING: SSH key files '/home/runner/.ssh/id_rsa' and '/home/runner/.ssh/id_rsa.pub' have been generated under ~/.ssh to allow SSH access to the VM. If using machines without permanent storage like Azure Cloud Shell without an attached file share, back up your keys to a safe location
ERROR: (UnsupportedAliasMinorVersion) 1.24 is not a supported Kubernetes version. To see our supported AKS version list please visit https://docs.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli for more details.
Code: UnsupportedAliasMinorVersion
Message: 1.24 is not a supported Kubernetes version. To see our supported AKS version list please visit https://docs.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli for more details.
Error: Process completed with exit code 1.
```

Example:
https://github.com/cilium/cilium/actions/runs/5825160549/job/15798796710

I have marked the ConformanceAKS job as "not required" for now on v1.12 branch until this is resolved.

--- a/.github/actions/azure/k8s-versions.yaml
+++ b/.github/actions/azure/k8s-versions.yaml
@@ -1,13 +1,13 @@
 # List of k8s version for AKS tests
 ---
 include:
-  - version: "1.24"
+  - version: "1.25"
     location: westeurope
     index: 1
-  - version: "1.25"
+  - version: "1.26"
     location: westus
     index: 2
-  - version: "1.26"
+  - version: "1.27"
     location: eastasia
     index: 3
     default: true

